python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -728,43 +720,6 @@ def test_groupby_unique_lists():
     )
 
 
-@pytest.mark.parametrize(
-    "data",
-    [
-        {"a": [], "b": []},
-        {"a": [2, 1, 2, 1, 1, 3], "b": [None, 1, 2, None, 2, None]},
-        {"a": [None], "b": [None]},
-        {"a": [2, 1, 1], "b": [None, 1, 0], "c": [None, 0, 1]},
-    ],
-)
-@pytest.mark.parametrize("agg", ["first", "last"])
-def test_groupby_first_last(data, agg):
```

Comment:
This removal _seems_ to decrease coverage because the none of the other tests cover `first` and `last` aggregations with Null column values (especially with nulls in first and last position). Is there a good reason to delete this test?

Reply:
That's a good point that I hadn't considered, and also highlights the fact that we aren't covering aggregations with null column values for the basic groupby tests. Given this, do you think it makes sense to:

- introduce null values to the `pdf` fixture being used for the basic groupby tests, and expand its use to `test_groupby_agg`
- add this test back in, modifying the name / parameters to specifically target the edge case where nulls are in the first and last position

Reply:
I think we want to test null and non-null dataframes, presumably for all the cases. So perhaps add an additional (via pytest.fixture dataframe) to the basic groupby tests.